### DISCUSSION TITLE
How to use Lazy Load with Picture Tag?

### DISCUSSION THREAD

**Discussion by gr0undhog** [nolimits4web/swiper#4542] [Q&A]
Hey Guys, 

Sorry I'm a noob. I tried to use lazyloading with picture-Tag and sourceset for webp and Jpgs. unfortunately it don't work for me. I have seen, that it should be fixed an May 2020, but it is not documented anywhere.

My code looks like this:

`<div class="swiper-container">`
  `<div class="swiper-wrapper">`
    `<div class="swiper-slide">`
      `<picture class="swiper-lazy">`
        `<source data-srcset="{$inhalt.imageWebp}" type="image/webp">`
        `<source data-srcset="{$inhalt.imageJpg}" type="image/jpeg" > `
        `<img data-src="{$inhalt.imageJpg}" alt="{$inhalt.imageAltText}"/>`
      `</picture>`
    `</div>`
  `</div>`
`</div>`

I am using the swiper.bundle.js in version 6.5.9

Can you tell me, where I got the mistake? 

Thanks!!

**Comment by gr0undhog** [comment]
Is there any solution or docu to this?

**Comment by memegauste** [comment]
You could write backend method that, using header from request, gives you correct link to image.
It's easily possible in django, you can even convert here images via sorl-thumbnail library :)

**Reply by gr0undhog** [reply]
I am using the slider in a given webshop source, so I cannot change or manipulate image paths. I thought, it is enough do link the images in a <picture> construct and swiper will load it automatically by getting into viewport.

**Comment by derWebdesigner** [comment]
I did add the _swiper-lazy_ class to each source and img which seems to work fine.
